router/notesRouter.js

```diff
@@ -0,0 +1,81 @@
+// Router Model to keep things easy and readable
+const express = require("express")
+const router = express.Router();
+const notes = require('../helpers/notesModel')
+
```

Comment:
You can complete the `Provide documentation for how to interface with your API.` stretch goal with no problems, by using [jsdocs](https://code.visualstudio.com/updates/v1_10#_auto-jsdoc-comments)